static/src/javascripts/projects/common/modules/experiments/ab-tests.js

```diff
@@ -20,6 +22,8 @@ export const concurrentTests: $ReadOnlyArray<ABTest> = [
 ];
 
 export const epicTests: $ReadOnlyArray<EpicABTest> = [
+    articlesViewed,
+    countryName,
```

Comment:
the google sheets tests will take precedence over these tests

Reply:
do we not have any global google sheets tests running right now then?

Reply:
we're going to disable most tests in the google spreadsheet